Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -0,0 +1,152 @@
+import defaultValue from "../../Core/defaultValue.js";
+import GltfLoader from "../GltfLoader.js";
+import ModelExperimentalUtility from "./ModelExperimentalUtility.js";
+import ModelExperimentalSceneGraph from "./ModelExperimentalSceneGraph.js";
+import Resource from "../../Core/Resource.js";
+import RuntimeError from "../../Core/RuntimeError.js";
+import when from "../../ThirdParty/when.js";
+
+/**
+ * A 3D model based on glTF, the runtime asset format for WebGL. This is
+ * a new architecture that is more decoupled than the older {@link Model}.
+ *
+ * This class is still experimental. glTF features that are core to 3D Tiles
+ * are supported, but other features such as animation are not yet supported.
+ *
+ * @constructor
+ *
+ * @param {Object} options Object with the following properties:
```

Comment:
@sanjeetsuhag I just noticed, this class has many more options that are not documented

Reply:
Note that some options are set from `Gltf3DTileContent.update()` not the constructor

Reply:
I am currently going with the approach of documenting what we're actually using in `ModelExperimental`. I wish we could remove the unused parameters, however, I can't because of `getModelClass`.